lib/client.js

```diff
@@ -5,6 +5,7 @@
  * Dependencies
  */
 
```

Comment:
We should decide on a 'private' pattern (like in `Service`) and use it across all our classes.